terraform/AddEndpoints.sh

```diff
@@ -8,7 +8,7 @@ protocol="https://"
 protocol+=$domain_name  
 echo $1   
 echo $protocol  
-jq --arg key $1 --arg value $protocol '.[ .[]|length ]  += { "NAME" :$key, "VALUE": $value}' ../endpoints/internal-dev/endpoints.json > justForthisUse.json
-cp -f justForthisUse.json ../endpoints/internal-dev/endpoints.json 
+jq --arg key $1 --arg value $protocol '.[ .[]|length ]  += { "NAME" :$key, "VALUE": $value}' ../endpoints/$APIGEE_ENVIRONMENT/endpoints.json > justForthisUse.json
```

Comment:
Not the highest priority for changes but, the KVM structure is the same for all environments. It's better if we construct the value (the jason payload) here in this script. Having a separate file per environment indicates that each file can contain different payloads but that's not the case.